src/main/java/org/springframework/data/elasticsearch/client/TransportClientFactoryBean.java

```diff
@@ -109,6 +111,7 @@ private Settings settings() {
 				.put("client.transport.ignore_cluster_name", clientIgnoreClusterName)
 				.put("client.transport.ping_timeout", clientPingTimeout)
 				.put("client.transport.nodes_sampler_interval", clientNodesSamplerInterval)
+				.put("shield.user", String.format("%s:%s", this.username, this.password))

```

Comment:
I think that since properties can already be specified (and should be exposed as something you can set via the namespace), we can already pass arbitrary properties to plugins hence I view this as unnecessary.